core/src/main/java/org/mule/runtime/core/internal/construct/DefaultFlow.java

```diff
@@ -39,17 +36,11 @@
 import reactor.core.publisher.Mono;
 
 /**
- * This implementation of {@link AbstractPipeline} adds the following functionality:
- * <ul>
- * <li>Rejects inbound events when Flow is not started</li>
- * <li>Gathers statistics and processing time data</li>
- * <li>Implements MessagePorcessor allowing direct invocation of the pipeline</li>
- * <li>Supports the optional configuration of a {@link ProcessingStrategy} that determines how message processors are processed.
- * </ul>
+ * Default implementation of {@link org.mule.runtime.core.api.construct.Flow}
  */
-public class Flow extends AbstractPipeline implements Processor {
+public class DefaultFlow extends AbstractPipeline implements org.mule.runtime.core.api.construct.Flow {
 
-  public Flow(String name, MuleContext muleContext) {
+  public DefaultFlow(String name, MuleContext muleContext) {
```

Comment:
Shouldn't this visibility be reduced to ideally force use of builder. This is what we did with Message/Event and others.